### DISCUSSION TITLE
Keda production grade

### DISCUSSION THREAD

**Discussion by yuvalweber** [kedacore/keda#3913] [Q&A / Need Help] [answered]
I searched for production grade document regarding Keda about which security requirements we should take like netowrkPolicies or something like that And which kind of metrics we should monitor and moreover.
Do you guys have some kind of consideration we should take?

**Comment by tomkerkhove** [comment, marked as answer]
We provide operational docs on https://keda.sh/docs/2.8/operate/

In terms of security requirements, we apply them automatically for you but always happy to hear what you need.

**Reply by yuvalweber** [reply]
Can you at least tell me what you mean regarding the security you implement. Cause for example I mean regarding maybe networkPolicies or something like this that I need to think before implementing @tomkerkhove

**Reply by tomkerkhove** [reply]
We don't require any network policies if that is what you are asking? These are the ports that are required to be usable: https://keda.sh/docs/2.8/operate/cluster/#firewall

And KEDA needs to be reachable by Kubernetes control-plane

**Reply by yuvalweber** [reply]
alright thanks 😄
